docs/posts.md

```diff
@@ -0,0 +1,54 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Posts
+
+<FeatureAvailability/>
+DataHub allows users to make Posts that can be displayed on the app. Currently, Posts are only supported on the Home Page, but may be extended to other surfaces on the app in the future. Posts can be used to accomplish the following:
```

Comment:
maybe change "surface on the app" to "surface of the app"?

Reply:
Makes sense